.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ccd-definition-processor"]
```

Comment:
rather than using a submodule we should use the docker image, I'll show you tomorrow